core/services/offchainreporting/validate.go

```diff
@@ -25,7 +26,7 @@ type ValidationConfig interface {
 }
 
 // ValidatedOracleSpecToml validates an oracle spec that came from TOML
-func ValidatedOracleSpecToml(config ValidationConfig, tomlString string) (job.Job, error) {
+func ValidatedOracleSpecToml(chainCollection evm.ChainCollection, tomlString string) (job.Job, error) {
```

Comment:
how about naming the type `evm.ChainSet` or `evm.Chains`?

Reply:
To my eye `Chains` is a bit similar to `Chain`. What don't you like about the name `ChainCollection`?

Reply:
Renamed to ChainSet